python.exe: can't open file '\\\\wsl.localhost\\Ubuntu\\mnt\\c\\Users\\Shaun Padrejuan\\Documents\\GitHub\\LabExercise3\\scripts\\customer_fix_naming_conventions.py': [Errno 13] Permission denied
